docs/index.html

```diff
@@ -88,13 +89,148 @@ <h2 id="getting-started">Getting started</h2>
 </ul></li>
 </ol>
 
-<p>At this point, if you haven&rsquo;t already, you should fire up some resources that Wash can actually introspect. Otherwise, as Han Solo would say, &ldquo;this is going to be a real short trip&rdquo;. So fire up some Docker containers, create some EC2 instances, toss some files into S3, launch a Kubernetes pod, etc.</p>
+<p>At this point, if you haven&rsquo;t already, you should fire up some resources that Wash can actually introspect. Otherwise, as Han Solo would say, &ldquo;this is going to be a real short trip&rdquo;. So fire up some Docker containers, create some EC2 instances, toss some files into S3, launch a Kubernetes pod, etc. We&rsquo;ve also provided a <a href="#wash-by-example">guided tour</a> that includes a Docker application.</p>
+
+<h3 id="known-issues">Known Issues</h3>
+
+<h4 id="on-macos">On macOS</h4>
+
+<p>If using iTerm2, we recommend installing <a href="https://www.iterm2.com/documentation-shell-integration.html">iTerm2&rsquo;s shell integration</a> to avoid <a href="https://github.com/puppetlabs/wash/issues/84">issue#84</a>.</p>
+
+<p>If the <code>wash</code> daemon exits with a exit status of 255, that typically means that <code>wash</code> couldn&rsquo;t load the FUSE extensions. MacOS only allows for a certain (small) number of virtual devices on the system, and if all available slots are taken up by other programs then we won&rsquo;t be able to run. You can view loaded extensions with <code>kextstat</code>. More information in <a href="https://github.com/osxfuse/osxfuse/issues/358">this github issue for <em>FUSE for macOS</em></a>.</p>
+
+<h2 id="wash-by-example">Wash by Example</h2>
```

Comment:
Headings should be sentence-style capitalization (throughout): Wash by example

Reply:
Same with `Known issues`?

Reply:
Yes. I didn't do a close read to find every instance, hence my lazy editor's "throughout" ;) Puppet style for headings is sentence style: Capitalize first word, the word after a colon, and proper nouns. Otherwise lowercase.